web-client/src/presenter/computeds/blockedCasesReportHelper.js

```diff
@@ -32,12 +33,28 @@ export const blockedCasesReportHelper = (get, applicationContext) => {
     blockedCasesFormatted = blockedCases
       .sort(applicationContext.getUtilities().compareCasesByDocketNumber)
       .map(blockedCase => {
+        const inConsolidatedGroup = !!blockedCase.leadDocketNumber;
```

Comment:
At this point, we are using this exact same logic in three helpers. You could extract this logic to a utility method and call that method from all three places to DRY up your code!